packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -725,10 +727,12 @@ export class RemoteMediaManager extends EventsScope {
   /**
    * Logs the state of the receive slots
    */
-  private logReceieveSlots() {
+  private logMainVideoReceiveSlots() {
     let logMessage = '';
     forEach(this.receiveSlotAllocations.activeSpeaker, (group, groupName) => {
-      logMessage += `group: ${groupName}\n${group.slots.map((slot) => slot.logString).join(' ')}`;
+      logMessage += `\ngroup: ${groupName}\n${group.slots
```

Comment:
i kinda hate that we add new lines into it. every log should be a singular line IMO as this makes the formatting much easier. instead, using a series of other delimiters would be good, but this is obviously outside the scope of this PR...

Reply:
yeah first time I saw these multiline logs I also thought that's a bad idea, but then it actually makes life easier, they're very easy to read this way so I'm not 100% convinced on this, we can discuss within the wider team and update in another PR